optuna/_dataframe.py

```diff
@@ -0,0 +1,83 @@
+import collections
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Set
+from typing import Tuple
+
+import optuna
+from optuna._imports import try_import
+from optuna.trial._state import TrialState
+
+
+with try_import() as _imports:
+    # `trials_dataframe` is disabled if pandas is not available.
```

Comment:
A nit, but `trials_dataframe` seems to be either `Study.trials_dataframe` or `_trials_dataframe`.
```suggestion
    # `Study.trials_dataframe` is disabled if pandas is not available.
```